contracts/ReflectionToken/ReflectionToken.sol

```diff
@@ -10,6 +10,9 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/utils/Address.sol";
 
 contract ReflectionToken is IReflectionToken, Ownable {
+    // Review: 
+    // - TODO: Should comment that fee is the numerator in case the denominator is maxFee.
```

Comment:
Sorry but I don't understand this line

Reply:
I mean, the unit of `anyFee` variable is the numerator correspond to the denominator of `maxFee`, then you should mention that. 
According to `_calculateFee()` function, fee variable seems like a numerator. So I mentioned this comment.
https://github.com/zkitty-norika/bunzz_module_audit/blob/norika/review-contract/contracts/ReflectionToken/ReflectionToken.sol#L1031-L1035

My point is the naming of `anyFee` is very confusing with the ariables of `FeeValues`. 
I cannot understand how they are different.

Reply:
Lol, the fee is percent value.
And in this contract, 10000 means 100%
So if fee is 500, that means 5 %.